src/transaction.rs

```diff
@@ -15,7 +15,6 @@ use alloc::Vec;
 use bigint::{U256, H256, Address, Gas};
 
 use super::errors::{RequireError, CommitError};
-#[cfg(feature = "std")]
 use super::errors::PreExecutionError;
```

Comment:
You could also move `PreExecutionError` to above errors import.

Reply:
Oops, right. Thanks!